src/static/bin/mailparse

```diff
@@ -55,6 +57,12 @@ def really_decode_header(value) -> str:
     return ''.join(decode_string(raw, encoding) for raw, encoding in values)
 
 
+def not_empty(data: T.Optional[str]) -> bool:
```

Comment:
wouldn't it be better to make an `empty()` function and then use `not empty(name)` instead of `not_empty(name)`? I have a hard time following too many negations!